app_server/views/serviceDashboard.pug

```diff
@@ -55,81 +55,264 @@ block content
                     table.table
                       thead
                         tr
-                          th Description
-                          th M
-                          th D
-                          th C
+                          th(style='width: 28%') Description
```

Comment:
This is in pure css but you might be able to do it in a cleaner way with bootstrap